### PR TITLE
mdspan PoC for distance make_blobs

### PR DESCRIPTION
**Pull request by cjnolet**

### CODE REVIEW

**Comment by cjnolet** [comment]
rerun tests

File: README.md [hunk 1]
```diff
@@ -32,38 +32,31 @@ The Python API is being improved to wrap the algorithms and primitives from the
 
 ## Getting started
 
-### Rapids Memory Manager (RMM)
```
Comment (dantegd) [inline README.md:35]: RMM is still used and required, no? I think a less prominent mention somewhere might still be good to have. Additionally, I think a quick summary of mdspan/array/etc. might be good to have for newcomers.

File: README.md [hunk 2]
```diff
@@ -32,38 +32,31 @@ The Python API is being improved to wrap the algorithms and primitives from the
...
-RAFT relies heavily on [RMM](https://github.com/rapidsai/rmm) which, 
-like other projects in the RAPIDS ecosystem, eases the burden of configuring different allocation strategies globally 
-across the libraries that use it. RMM also provides [RAII](https://en.wikipedia.org/wiki/Resource_acquisition_is_initialization)) wrappers around device arrays that handle the allocation and cleanup.
-
 ### C++ Example
 
 Most of the primitives in RAFT accept a `raft::handle_t` object for the management of resources which are expensive to create, such CUDA streams, stream pools, and handles to other CUDA libraries like `cublas` and `cusolver`.
 
-The example below demonstrates creating a RAFT handle and using it with RMM's `device_uvector` to allocate memory on device and compute
+The example below demonstrates creating a RAFT handle and using it with `device_matrix` and `device_vector` to allocate memory, generating random clusters, and computing
```
Comment (dantegd) [inline README.md:39]: Another documentation that would be good to have is the fact that both old and new API are present but the new one is the way to go (and old one eventually will be deprecated).

Comment (dantegd) [inline README.md:39]: Another documentation that would be good to have is the fact that both old and new API are present but the new one is the way to go (and old one eventually will be deprecated).

**Review by dantegd — CHANGES_REQUESTED** [review]
The changes look fantastic, just added comments about additional documentation

**Review by dantegd — APPROVED** [review]

**Comment by cjnolet** [comment]
@gpucibot merge

**Pull request merged**